mmv1/products/cloudbuild/Trigger.yaml

```diff
@@ -1118,6 +1118,7 @@ properties:
               - :N1_HIGHCPU_32
               - :E2_HIGHCPU_8
               - :E2_HIGHCPU_32
+              - :E2_MEDIUM
```

Comment:
Instead of adding a new value, it is recommended to change the type from Enum to String, as `machineType` changes frequently.